apps/src/code-studio/components/progress/MiniViewTopRow.jsx

```diff
@@ -47,11 +47,18 @@ export default class MiniViewTopRow extends React.Component {
 
   render() {
     const {scriptName, linesOfCodeText, selectedSectionId} = this.props;
-
     const sectionId = queryParams('section_id');
-    const params = selectedSectionId
-      ? stringifyQueryParams({section_id: selectedSectionId})
-      : stringifyQueryParams({section_id: sectionId});
+    var params;
+    switch (true) {
+      case typeof selectedSectionId !== 'undefined' && selectedSectionId !== '':
+        params = stringifyQueryParams({section_id: selectedSectionId});
+        break;
+      case typeof sectionId !== 'undefined' && sectionId !== '':
```

Comment:
I would suggest changing these to:
```
case !!selectedSectionId:
case !!sectionId:
```
to simplify this logic, unless there is a falsy value we'd want to allow through.